klimenko/src/com/company/Main.java

```diff
@@ -0,0 +1,13 @@
+package com.company;
+
+import java.util.Arrays;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Integer[] array = {10, 5, 30, 15, 36, 88, 23};
+        System.out.println(Arrays.asList(array).stream().max(Integer::compareTo).get());
```

Comment:
Can you write it manually? Method, which takes array as argument and returns max value?